src/assertions/map.rs

```diff
@@ -66,6 +67,13 @@ where
         K: Eq + Hash + Debug,
         V: Eq + Debug;
 
+    /// Checks that the subject contains all entries from `expected`.
+    fn contains_all<BM>(&self, expected: BM) -> R
```

Comment:
Could you rename to `contains_at_least` to follow naming style of Java Truth?

https://truth.dev/api/latest/com/google/common/truth/MapSubject.html#containsAtLeast(java.lang.Object,java.lang.Object,java.lang.Object...)